SplitFiles/game.c

```diff
@@ -82,8 +82,9 @@ int main_game()
     printf(PRINTAT "\x01\x07" "\x12\x31\x10\x32" "Can you handle the pressure?\x12\x30\x10\x30");
     printf(PRINTAT "\x05\x0B" "Press any key to start");
     wait_for_a_key(NULL, NULL);
-    int rando = *((int *)49161);
-    srand(rando);
+    // get a random seed based on frame count
+    int *rando = 23672;
```

Comment:
I don't think this is going to work as you're hoping, not with startup=1 anyway.  I don't see the frame count being updated when I write test code to print it out.  In the three positions for the timer I instead see 22, 0, 0 - I suspect 22 is the frame count needed before the C runtime takes over the machine...

We could check the keyboard asynchronously in a loop until the users presses a key - count the iterations and use that as the seed?

Reply:
I did test it and it seemed to work

Reply:
let me check again

Reply:
yes, that's possible. But that doesn't matter except for the fact that the seed for the first game will always be the same unfortunately.

Reply:
I can do your key thing though